### PR TITLE
Add custom configuration command to allow adding new language dictionary 

### PR DESCRIPTION
**Pull request by sylviamoss**
Usage: 
```
diceware config --add --lang --source=/Users/sylviamoss/Projects/diceware-cli/test.txt --name=es
```
or
```
diceware config -a -l -s=/Users/sylviamoss/Projects/diceware-cli/test.txt -n=es
```

Help doc
```
Adds diceware custom configuration such as new languages

Usage:
  diceware config [flags]

Flags:
  -a, --add             add new config
  -h, --help            help for config
  -l, --lang            add new language
  -n, --name string     language name
  -s, --source string   dictionary source file
```

Closes https://github.com/sylviamoss/diceware-cli/issues/8
Closes https://github.com/sylviamoss/diceware-cli/issues/2

### CODE REVIEW

File: cmd/config.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	customConfig diceware.CustomConfig
+
+	configCmd = &cobra.Command{
+		Use:   "config",
+		Short: "Adds diceware custom configuration such as new languages.",
```
Comment (vrcca) [inline cmd/config.go]:
```suggestion
		Short: "Adds a new language.",
```

File: diceware/custom_config.go [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+		return err
+	}
+	defer file.Close()
+
+	home, err := os.UserHomeDir()
+	if err != nil {
+		return err
+	}
+
+	dicewarePath := home + "/.diceware/diceware_words_" + c.Name
```
Comment (vrcca) [inline diceware/custom_config.go]: What happens if the user passes an invalid name? Eg. (Windows): `pt:BR`

**Comment by vrcca** [comment]
Thanks! It looks great!

File: diceware/custom_config.go (hunk 2 above)
Comment (sylviamoss) [reply diceware/custom_config.go]: now it will fail if it's not valid 👀

**Pull request merged**